tests/integration/chunkstore/test_chunkstore.py

```diff
@@ -537,7 +537,7 @@ def write_random_data(chunkstore_lib, name, month, days, securities, chunk_size=
     month: integer
     days: list of integers
     securities: list of integers
-    chunk_size: one of 'D', 'M', 'Y'
+    chunk_size: one of 'D', 'M', 'A'
```

Comment:
Yeah, but I think people would guess 'DMY', rather than 'DMA'.  Nearly everywhere else we talk about years rather than annos.